src/TestReporter/Entity/JsonFile.php

```diff
@@ -85,14 +94,19 @@ protected function collectSourceFiles()
     }
 
     /**
-     * @param SourceFile $sourceFile
+     * @param CoverallsV1Bundle\Entity\SourceFile $sourceFile
      * @return string
      */
-    protected function calculateBlobId(SourceFile $sourceFile)
+    protected function calculateBlobId(CoverallsV1Bundle\Entity\SourceFile $sourceFile)
     {
         $content = file_get_contents($sourceFile->getPath());
         $header  = "blob " . strlen($content) . "\0";
 
         return sha1($header . $content);
     }
+
+    public function __toString()
```

Comment:
Not sure what you think, but personally, I would go as far as reducing the visibility of all other methods in this class (except for the constructor) to `private` - none of them are used outside the context of this class.